docs/Introduction/architecture-overview/architecture-decentralized-model.md

```diff
@@ -92,12 +92,12 @@ Learn how to create a consumer contract to [Get the Latest Price](../get-the-lat
 
 Proxy contracts are on-chain proxies that store the most up-to-date Aggregator for a particular price feed. Using proxies enables the underlying Aggregator to be upgraded without any interruption of service for consuming contracts.
 
-See the <a href="https://github.com/smartcontractkit/chainlink/blob/master/evm-contracts/src/v0.6/AggregatorProxy.sol" target="_blank">`AggregatorProxy`</a> contract on Github.
+See the <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/dev/AggregatorProxy.sol" target="_blank">`AggregatorProxy`</a> contract on Github.
```

Comment:
Shouldn't we still point at 0.6?

Reply:
Yes but there's not AggregatorProxy for 0.6 that's why I picked 0.7. I can change to another interface if needed.

Reply:
Hrm. Can you try to figure out why we removed the 0.6 contract? That's unusual right?